feature_test.rb

```diff
@@ -0,0 +1,27 @@
+2.2.3 :001 > g = Game.play
```

Comment:
Might need to walk this through with me? 
Is this just a feature test, to test the app in irb?